README.md

```diff
@@ -22,65 +86,113 @@ For the latter you will need:
 - bundler (see [bundler installation](https://bundler.io/); for version, see [.bundler-version](./.bundler-version))
 - yq (see [yq installation](https://github.com/mikefarah/yq#install))
 
-### Windows support
+#### Windows support
 
 We provide an experimental support for Window 10.
 
 You can run the `win32\install.bat` script to have an unattended installation of Ruby
 and the tools we use. Once installed, you can run the `specs` using `make.bat`
 
-### Running the connector
+### Implementing the connector protocol using the connector framework
+
+The first three steps for this would be the same [as for skipping the framework and implementing the protocol manually](#implementing-the-connector-protocol): creating a document index, setting up an API key, and updating the document index mappings as needed. The rest of the steps, however, are different.
+
+- Copy the [example connector](lib/connectors/example/connector.rb) into a separate folder under the [connectors](lib/connectors). Rename the class as required.
+- Change `self.service_type` and `self.display_name` to match the connector you are implementing.
+- Change `self.configurable_fields` to provide a list of fields you want to configure via the Kibana UI on connecting the connector.
+- Implement the `health_check` method to return a boolean value, corresponding to the health status of the connector. You should return `true` if the third-party data source is working and `false` if it is not.
+- Implement the `yield_documents` method to yield documents to the connector framework. It should call `yield` for every document separately and you should yield the documents in the format, matching the mappings that you have defined for the documents index.
 
-TBD
+### Operating the connector service
```

Comment:
The current tone here is "how it works". Ideally, this would be presented more as "what to do". For example:

- Within Kibana, navigate to...
- Enter the values for...

Reply:
So it's where we should specify how to actually connect the connector?

Reply:
Yes? I think I intended for this to be the part of the outline where the role switches from developer to service operator, so now you're actually _using_ the connector. Although for 8.4, the developer and operator are likely the same person. In future releases we should separate out dev docs and usage docs, but for this release I don't think it's as important.

Reply:
Gotcha, changing it.